tidb-cloud/monitor-tidb-cluster.md

```diff
@@ -26,22 +26,29 @@ You can see the current status of each running cluster on the cluster page.
 
 ### TiDB node status
```

Comment:
Please modify L14 `### Cluster Status` to `### Cluster status`.

Reply:
done